layer/flavors/custom.py

```diff
@@ -44,12 +46,43 @@ def save_model_to_directory(
         directory: Path,
     ) -> None:
         directory.mkdir(parents=True, exist_ok=True)
+
+        # Store class config
+        with open(directory / self.MODEL_CONFIG_FILE, mode="w") as file:
+            custom_model_class = type(model_object)
+            class_config = custom_model_class.__module__ + "." + custom_model_class.__name__
+            file.write(class_config)
+
+        # Store class definition
+        import inspect
+        custom_class_model = type(model_object)
+        class_model_source_code = inspect.getsource(custom_class_model)
```

Comment:
Can we add a check to make sure we are getting the source or throw an error so that user needs to define it in another file ? How about notebooks will it work on that case ?

Reply:
Ok, adding checks. 

Yes it works in the notebook case with the only limitation to implement the custom model in a seperate python file.